tests/cover/test_direct_strategies.py

```diff
@@ -252,7 +252,8 @@ def test_decimals():
 
 
 def test_non_float_decimal():
-    find(ds.decimals(), lambda d: float_to_decimal(float(d)) != d)
+    find(ds.decimals(),
+         lambda d: not d.is_snan() and float_to_decimal(float(d)) != d)
```

Comment:
Because `float(Decimal('snan'))` raises an exception, and this test finds 0.3333... (>float precision).  So I restored it to the behavior decimals() had when the test was written to stop unrelated failures in CI.